lib/playground.dart

```diff
@@ -203,9 +203,14 @@ class Playground {
       _handleHelp();
     });
 
+    keys.bind(['alt-enter', 'ctrl-1'], (){
+      if (editor.document.hasIssueAtOffset) {
```

Comment:
If they hit ctrl-1, should we always look for quick-fixes? I don't think we need the conditional here.